Buildings.Json

```diff
@@ -7,6 +7,12 @@
                 "greatPersonPoints": {"Great General": 1}
                 "science": 2,
                 "culture": 1,
-		"uniques": ["New [Military] units start with [25] Experience [in this city]", "Destroyed when the city is captured"], "[5] units cost no maintenance", "[2] free [Musketmen] units appear","Requires at least [12] population"
+		"uniques": [
+			"New [Military] units start with [25] Experience [in this city]",
+			"Destroyed when the city is captured",
+			"[5] units cost no maintenance", //Are you aware of this stacking for multiple cities so you effectively pay no maintenance at all?
```

Comment:
Yes. That’s why I made it so that a city has to be at 12 pop or above to make it, the building have a higher maintenance, and give 2 free musketmen. This way, it counts as far less free units and makes it so that you still pay much.

Reply:
Thanks for the syntax fix!